src/main/resources/messages.properties

```diff
@@ -0,0 +1,52 @@
+widget.meta.title=Mit BundesIdent online ausweisen
```

Comment:
Is there a way to let Spring use `de` as default language without a need for duplicating this file? 🤔

Reply:
Good point. I mean we could just remove the `_de` file. This will have the desired effect that we do not have duplicate files and german is the default.

Reply:
That's fine for me for now! 👍

Reply:
Done.